docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '2.1'
+
+services:
+  zoo1:
+    image: zookeeper:3.4.9
```

Comment:
NABD, but there's 3.5.5 if you want the latest.

Reply:
Don't know why, but when changed to the newest version, this error accures:

kafka1_1  | [main-SendThread(zoo1:2181)] INFO org.apache.zookeeper.ClientCnxn - Opening socket connection to server zoo1/172.19.0.2:2181. Will not attempt to authenticate using SASL (unknown error)
kafka1_1  | [main-SendThread(zoo1:2181)] INFO org.apache.zookeeper.ClientCnxn - Socket error occurred: zoo1/172.19.0.2:2181: Connection refused